pkg/command/util/consts.go

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022 Singularity Data
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package util
+
+const (
+	COMPUTE      = "compute"
+	META         = "meta"
+	FRONTEND     = "frontend"
+	COMPACTOR    = "compactor"
+	GLOBAL       = "global"
```

Comment:
It's better to follow the Go's style and use the CamelCase for constants, unless we have some problem with representing the meanings or it's just imported from C-style libs. Please revise these consts.